Sources/EdgeConstants.swift

```diff
@@ -42,6 +42,7 @@ enum EdgeConstants {
         static let REQUEST_EVENT_ID = "requestEventId"
         static let DATASET_ID = "datasetId"
         static let CONSENTS = "consents"
+        static let EXTENSIONS = "extensions"
```

Comment:
This should probably be under EdgeConstants.SharedState.EventHub.EXTENSIONS as it is not part of any event data but rather Event Hub's shared state